source/sourcing_and_processing_data/processing.rst

```diff
@@ -1349,35 +1327,138 @@ It takes the following parameters:
     * Field
     * yes
 
-Simplify Geo Shape
-~~~~~~~~~~~~~~~~~~
+GeoJoin
+~~~~~~~
 
-Simplify a geo shape to reduce processing time and dataset size.
+This processor collection retrieves administrative divisions **Geo Shapes** from a key (postcode, county code, etc.). Each country has a dedicated processor and specific referentials.
+These referentials, which are referenced in the tables below, can be found in datasets available on https://public.opendatasoft.com.
```

Comment:
In my opinion, it would be better if there was a link to https://public.opendatasoft.com.